bcgov/mds
# Adding lead and issuing inspector name column in the Applications page.

# Main
Add Lead and Issuing Inspector columns in Application page.

# Other
- N/A

# How to test
- Under Permit & Approvals select Applications for a specific mine:
- Two new columns are shown under the Notice of Work Applications:
![image](https://user-images.githubusercontent.com/10526131/137411666-76b57118-a512-44bb-91df-e024dbc494dc.png)

# Notes
https://bcmines.atlassian.net/browse/MDS-3806
